src/seam-connect/client.ts

```diff
@@ -45,9 +45,11 @@ export const getSeamClientOptionsWithDefaults = (
 
   return {
     apiKey:
-      providedOptions.apiKey ??
-      globalThis?.process?.env?.SEAM_API_KEY ??
-      undefined,
+      providedOptions.apiKey === null
+        ? null
+        : providedOptions.apiKey ??
+          globalThis?.process?.env?.SEAM_API_KEY ??
+          undefined,
```

Comment:
This or the other suggestion so it's consistent. Probably the other one.
```suggestion
          null,
```